scripts/quote-board.coffee

```diff
@@ -0,0 +1,42 @@
+# Description:
+#  Quote Board Interface
+#
+# Dependencies:
+#  None
+#
+# Commands:
+#  !quote <name> said <quote> - Adds that quote attributed to that person to the DB
+#  !quote_me_as <name> - Adds <name> to be your full name for quotes attributed to you
+#
+# Notes:
+#  None
+#
+# Author:
+#  @murph33
+
+QUOTE_URL = "https://quote-board.herokuapp.com/api/v1/quotes"
+PERSON_URL = "https://quote-board.herokuapp.com/api/v1/people"
```

Comment:
I would recommend pulling these into env variables like the other scripts, for easier development.